lib/base/Version.js

```diff
@@ -31,8 +30,15 @@ Version.prototype.absoluteUrl = function (uri) {
  * @param  {string} uri uri to transform
  * @return {string} transformed url
  */
-Version.prototype.relativeUrl = function (uri) {
-  return _.trim(this._version, '/') + '/' + _.trim(uri, '/');
+Version.prototype.relativeUrl = function(uri) {
+  const trim = (str, c = '\\s') => str.replace(new RegExp(`^([${c}]*)(.*?)([${c}]*)$`), '$2');
+  if (typeof this._version !== 'string') {
+    if (uri === undefined) {
+      return '';
+    }
+    return trim(uri, '/');
```

Comment:
The original implementation relied on lodash's `trim()` returning empty strings for undefined/null input, will fix it up though

Reply:
Just mean if the first part was null/undefined then the concatenation would have a leading `/`, and if the second part was null/undefined then it would end with a trailing `/`. Not sure if this is important.

Reply:
Ah gotcha, fixed